go/vt/vtctl/vtctl.go

```diff
@@ -1832,6 +1833,7 @@ func commandCreateKeyspace(ctx context.Context, wr *wrangler.Wrangler, subFlags
 	baseKeyspace := subFlags.String("base_keyspace", "", "Specifies the base keyspace for a snapshot keyspace")
 	timestampStr := subFlags.String("snapshot_time", "", "Specifies the snapshot time for this keyspace")
 	durabilityPolicy := subFlags.String("durability-policy", "none", "Type of durability to enforce for this keyspace. Default is none. Possible values include 'semi_sync' and others as dictated by registered plugins.")
+	sidecarDBName := subFlags.String("sidecar-db-name", sidecardb.DefaultName, "Vitess sidecar database used for internal metadata.")
```

Comment:
We might want to consider the magic value `"auto"` to auto-generate a sidecar db name, e.g. if my keyspace is `commerce` then the auto generated name would be `_vt_commerce`.